Makefile

```diff
@@ -77,6 +78,9 @@ deploy-image:
 deploy: deploy-setup image deploy-image
 	hack/deploy.sh
 
+deploy-example: deploy
+	oc create -n $(NAMESPACE) -f hack/cr.yaml
```

Comment:
`hack/cr.yaml` is missing the managed spec field, `hack/examples/cr.yaml` has it though.
I'm find with whichever one we use but we should ensure it has that spec and lets remove the dupe